### ISSUE TITLE
Minimal required features for the Project Pythia landing page

### ISSUE THREAD

**Issue by brian-rose** [projectpythia-mystmd/projectpythia-mystmd.github.io#1]
Since this repo houses the experimental source for a potential new landing page / web site for Project Pythia, let's track here the missing features from our [existing Sphinx-based home page](https://projectpythia.org) that are considered essential before migrating to MyST-MD. 

We are comparing this [new landing page](https://projectpythia-mystmd.github.io) to the [existing landing page](https://projectpythia.org).

I'll take a first stab at a minimal feature list:

- [x] Landing page is clean and simple and less "bookish"
- [x] "call to action" buttons are appropriately sized and styled
- [x] Pythia google calendar is displayed #7 
- [x] Admonitions display properly, e.g. on [the Contributor guide page](https://projectpythia-mystmd.github.io/contributing)
- [x] [Code of Conduct page](https://projectpythia.org/code_of_conduct.html) is findable
- [x] All remaining "Unknown Directive" errors are cleaned up
- [x] A branded banner image
- [ ] Blog posts are easily findable
- [ ] Footer with essential credits (https://github.com/jupyter-book/myst-theme/pull/565)

What else? What did I miss?

There is some overlap here with the checklist over at https://github.com/executablebooks/mystmd/issues/1320#issue-2354030697, but I'm trying to keep this list specific to the main "portal" site and landing page.

**Comment by clyne** [comment]
Looks pretty complete to me, @brian-rose

**Comment by agoose77** [comment]
@jukent I see that you authored https://github.com/ProjectPythia/projectpythia.github.io/pull/412, which added metrics to the landing page! Before seeing the existing implementation, my first thought was to implement the metrics page as an executable file and build the figures inline. Is there a particular reason that you chose to do this via a dedicated job in CI?

**Comment by jukent** [comment]
@agoose77 can you clarify how you would have done it? I did it in CI because that's how I knew to make sure the metrics were updated and populated before the book build and publish actions. I'm definitely open to a different method in the MyST version and would love to learn more.

**Comment by agoose77** [comment]
@jukent yes of course! I should have clarified that part of my original message, apologies!

If the goal is to generate content dynamically, you can use the Jupyter integration with Jupyter Book 1 (and JB2 / MyST-MD) to run e.g. Python code in a notebook.

In MyST Markdown, you can add details about the kernel that should run the document, and then use code cells to build outputs like figures. I can make a PR that implements this approach, and perhaps pull you in to review it?

**Comment by agoose77** [comment]
> Are you able to pass GitHub secrets [...] into the code

My thought would be to do this through the environment. There is a risk there if someone commits a notebook that dumps the environment variables to stdout, but you already maintain control over the repository contents, so that should be fine?

**Comment by brian-rose** [comment]
> [@brian-rose](https://github.com/brian-rose) what constitutes "branded banner image" to you?
> 
> Does this come close?
> 
> ![Image](https://github.com/user-attachments/assets/44256ae0-eaeb-4699-9618-2cf30c8a398f)

Yes!

I think I was just broadly referring to the need to have a "landing page" look with some imagery. I think we can check off this box.

**Comment by jukent** [comment]
@agoose77 This reminded me that we were talking about adding an automatic metrics page to each Cookbook, so that Cookbook authors could see the reach of their work. If you'd like to look at one with me (say for the cookbook-template in MyST) that could be a smaller push than the portal metrics.

**Comment by jukent** [comment]
> [@brian-rose](https://github.com/brian-rose) could you help me to refine the meaning of
> 
> > blog posts are easily findable

I think having them in the navbar works, but ideally we'd like to improve upon the current portal. One idea we had was adding a mini-gallery of blog posts (sort of like in the [GeoCAT website](https://geocat.ucar.edu/)).

**Comment by brian-rose** [comment]
Yes I think our current landing page *only barely* meets this criterion with the "Blog" link in the top-most nav bar. 

I would love to a see a prominent "What's new" or "Recent news" section on the landing page that dynamically displays a few most recent blog posts. We've been compensating for the lack of this feature by manually adding banners for time-sensitive info (e.g. "Cookoff 2025"). A proper blog feed would obviate the need for these manual banners and buttons most of the time.

**Comment by agoose77** [comment]
I'm hoping to start working on the concept of a gallery that embeds card-like previews of resources (like MyST sites, or opengraph-enabled links). It sounds like that would fit the requirements of the resource and cookbook galleries, as well as a latest-blog-posts feature.

I need to touch base with the MyST team to discuss some of this, but I'll return here to update and iterate for the next sprint (later this week).

**Comment by jukent** [comment]
Thanks @agoose77, I am currently editing the MyST Cookbook gallery (locally) to attempt to ad filtering. I've added HTML/CSS classes to the cards, but am working on creating a dropdown menu that is populated with all of the tags. Will touch base with you when you work on the gallery.

**Comment by jukent** [comment]
Some things we noticed in today's Pythia sprint - the bottons on the landing page are not adaptable to mobile (probably just missing a container) 

![Image](https://github.com/user-attachments/assets/a6b8e56c-1782-4fb4-94bb-783fc9358a39)

And links to other pages (i.e. Contributor's guide) go to the GitHub page. Do we want to have another internal webpage that is themed consistently and to test that out in our "minimum features" checklist?

<img width="1294" alt="Image" src="https://github.com/user-attachments/assets/3ac8aeaa-6c79-4a28-8944-d6f4929a1426" />

**Comment by agoose77** [comment]
Thanks Julia!

The buttons have been fixed upstream!

We just need to make a release.
I'll get on that.
